packages/react-core/src/components/Select/SelectGroup.tsx

```diff
@@ -20,11 +20,11 @@ export const SelectGroup: React.FunctionComponent<SelectGroupProps> = ({
   titleId = '',
   ...props
 }: SelectGroupProps) => (
-  <div {...props} className={css(styles.selectMenuGroup, className)}>
-    <div className={css(styles.selectMenuGroupTitle)} id={titleId} aria-hidden>
-      {label}
-    </div>
-    {children}
-  </div>
+  <li {...props} className={css(styles.selectMenuGroupTitle, className)} id={titleId}>
+    {label}
+    <ul className={css(styles.selectMenuGroup)} role="listbox">
```

Comment:
I think you should only change that `children` is wrapped by `<ul className={css(styles.selectMenuGroup)} role="listbox">`, and leave the outer elements the way they were

Reply:
That's what the issue description suggests to do, but if you read the specification of what can be child of UL is not contain divs.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul

SelectGroups is direct child of UI, and should not contain any divs in other words.

Reply:
The DOM should look similar to this (copied from Core example):
<img width="790" alt="Screen Shot 2020-10-01 at 1 43 16 PM" src="https://user-images.githubusercontent.com/869106/94844443-3e75ba00-03ec-11eb-83d9-d828f421d7fa.png">

Reply:
Also the changes break the styling a bit, additional indentation etc
https://patternfly-react-pr-4919.surge.sh/components/select#grouped-checkbox-input

Reply:
Ack I see it now in the HTML docs. I will adjust it tomorrow.
Thanks for the review.

Reply:
I wonder if it's a concious decision though that nested ul li were not used in the HTML docs.

For me it makes sense that the outer items would be also a list (ul) since there indeed part of a list.

@mcarrano I guess you can answer this one :)  For example this https://patternfly-react-pr-4919.surge.sh/components/select/#grouped-single

Reply:
@mcoker can you answer the question above?

Reply:
@jschuler is right with the screenshot posted above. I left the same basic comment here - https://github.com/patternfly/patternfly-react/pull/4919#discussion_r498446371

FWIW you can find the same basic structure in the dropdown, or any menu with groups/titles https://www.patternfly.org/v4/components/dropdown/react/with-groups/

```
div/section.section
    div/h1-h6.section-title
    ul
        li
        li
        ....
```